src/bin/stratisd.rs

```diff
@@ -35,6 +38,29 @@ fn print_err(err: StratisError) -> () {
     eprintln!("{}", err.description());
 }
 
+/// Given a udev event check to see if it's an add and if it is return the device node and
+/// device number.
+fn handle_udev_add(event: &libudev::Event) -> Option<(PathBuf, u64)> {
```

Comment:
Recommend converting device.devnum() to a devicemapper::Device asap, and then returning that instead of `u64`. (Device has a from(u64, aka libc::dev_t) method.) Change `block_evaluate` to take a `Device`, too.